visualization/package.json

```diff
@@ -45,7 +45,7 @@
 	"scripts": {
 		"start": "nw .",
 		"build": "webpack --config ./conf/webpack.config.js --env DEV=false",
-		"dev": "webpack serve --config ./conf/webpack.config.js --env DEV=true",
+		"dev": "NODE_OPTIONS=--openssl-legacy-provider webpack serve --config ./conf/webpack.config.js --env DEV=true",
```

Comment:
Do we need this option? I cannot run the dev mode anymore.

Reply:
It is needed to run the project using 18.7.0 but given that the recommended version is 16 I removed the change